app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from typing import List
+
+from app.players.elves.elf import Elf
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
+    if not team:
+        return 0
```

Comment:
There is no need for this code, `sum` will return 0 for an empty list. Remove these lines.